lib/rules/space-infix-ops.js

```diff
@@ -27,33 +27,50 @@ module.exports = function(context) {
                 OPERATORS.indexOf(op.value) >= 0 &&
                 (tokens[i - 1].range[1] >= op.range[0] || op.range[1] >= tokens[i + 1].range[0])
             ) {
-                return false;
+                return op;
             }
         }
-        return true;
+        return null;
     }
 
-    function report(node) {
-        context.report(node, "Infix operators must be spaced.");
+    function report(mainNode, culpritNode) {
```

Comment:
Add some JSDoc here. Is `culpritNode` a node or a token? I think a token, probably should be `culpritToken`.